docs/user/document/distribution/tlp-usage.en.md

```diff
@@ -1 +1,11 @@
-# Tlp - Usage
+# TLP - Usage
+
+The Traffic Light Protocol (TLP) is a standardized agreement for the exchange of information worthy of protection
+(see [en.wikipedia.org/wiki/Traffic_Light_Protocol](https://en.wikipedia.org/wiki/Traffic_Light_Protocol)).
```

Comment:
I would use the link to https://www.first.org/tlp/docs/tlp-v1.pdf instead.

Reply:
updated